packages/react-reconciler/src/ReactFiberFlags.js

```diff
@@ -63,6 +120,7 @@ export const PassiveStatic = /*                */ 0b000100000000000000000000000;
 export const PlacementDEV = /*                 */ 0b001000000000000000000000000;
 export const MountLayoutDev = /*               */ 0b010000000000000000000000000;
 export const MountPassiveDev = /*              */ 0b100000000000000000000000000;
+export const MountInsertionDev = /*           */ 0b1000000000000000000000000000;
```

Comment:
I've added a new flag here. If this seems okay, I can change the other flags so their length matches this one (currently the rest are one shorter).